client/src/components/new-request/send-to-examination.vue

```diff
@@ -13,16 +13,13 @@
         <NameInput :is-read-only="true" class="mb-n2"/>
       </v-row>
 
-      <v-row no-gutters justify="center">
+      <v-row no-gutters justify="center" class="text-center">
         <v-col cols="auto" class="h5">
           <v-icon class="action-icon" :class="actionConfig.class">{{ actionConfig.icon }}</v-icon>
-          &nbsp;
           <span :class="actionConfig.class">{{ actionConfig.text }}</span>
         </v-col>
-        <v-col cols="12" class="py-2">
-          <p class="text-center copy-normal">
-            This name cannot be analyzed here and must be sent to staff for review.
-          </p>
+        <v-col cols="12" class="pb-2">
```

Comment:
Would it be better to have pb-0 on the p element below?
(Negative margins/padding are always a red flag.)

Reply:
Ok, will check it out!

Reply:
Fixed